### PR TITLE
[CHUX-242] Remove ellipsis on table filters

### PR DESCRIPTION
**Pull request by stavros-tomas**
Removing the max width limitations, if we do not like it in the future we can revert back to ellipsis or think of a better solution like inline the filters.

### CODE REVIEW

**Review by raulwwq0 — APPROVED** [review]

File: src/components/ec-filter-popover/ec-filter-popover.vue [hunk 1]
```diff
@@ -111,9 +111,6 @@ function onOpen(isOpen: boolean) {
 
   &__label {
     @apply tw-text-gray-3;
-    @apply tw-truncate;
```
Comment (raulwwq0) [inline src/components/ec-filter-popover/ec-filter-popover.vue:114]: Just one thing, I don't think we should allow the text from the filters to wrap, it should be in the same line

image

Comment (stavros-tomas) [reply src/components/ec-filter-popover/ec-filter-popover.vue:114]: I don't think we should restrict it. It's unnatural to don't allow the element to wrap, fold etc. 
If you don't allow them to do it and you also remove the ellipsis,  then will be even worst they will overflow.

If we see that they wrap then we need to do something in the UI like remove a filter or so.

Comment (raulwwq0) [reply src/components/ec-filter-popover/ec-filter-popover.vue:114]: Fine by me, but I would ask Alba anyways just in case...

Comment (stavros-tomas) [reply src/components/ec-filter-popover/ec-filter-popover.vue:114]: I asked Alba if she still wants to remove the ellipsis (which i think is the correct way to do this)

Comment (stavros-tomas) [reply src/components/ec-filter-popover/ec-filter-popover.vue:114]: We will go with this solution, if we don't like we can revert back to ellipsis or do some more major redesign of the header.

**Review by raulwwq0 — APPROVED** [review]

**Pull request merged**